br/pkg/aws/ebs_test.go

```diff
@@ -14,28 +14,60 @@
 package aws
 
 import (
+	"context"
 	"testing"
 
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/pingcap/tidb/br/pkg/gluetidb"
 	"github.com/stretchr/testify/require"
 )
 
+func str2Ptr(str string) *string {
```

Comment:
maybe use `aws.String` instead of this function.

Reply:
fix in https://github.com/pingcap/tidb/pull/39975/commits/6017b17dd8e1df611ba031638a6598359d2c8edf